manifests/charts/base/files/gen-istio-cluster.yaml

```diff
@@ -3500,6 +3500,9 @@ rules:
   - apiGroups: ["config.istio.io", "security.istio.io", "networking.istio.io", "authentication.istio.io"]
     verbs: ["get", "watch", "list"]
     resources: ["*"]
+  - apiGroups: ["networking.istio.io"]
+    verbs: [ "get", "watch", "list", "update", "create", "delete" ]
+    resources: [ "workloadgroups" ]
```

Comment:
Shouldn't we be updating workload entry (represents a single VM, which we declare healthy/unhealthy) rather than workload group (a template for the workload entry)?

Reply:
I guess that makes sense. I thought that since we configured Readiness Probe in Workload Group we would update status there as well.

Reply:
shouldnt it be both then? update status with `WorkloadEntry` but if the user wants to update the readiness probe configuration then they can update `WorkloadGroup`?

Reply:
The user will update WorkloadGroup though, not Istiod. We only need to add permission here for things istiod writes

Reply:
Also I am not 100% sure but it may need `workloadentry/status` as the resource, not just `workloadentry`

Reply:
that sounds about right